lib/mayaUsd/fileio/shading/shadingModePxrRis.cpp

```diff
@@ -415,6 +432,16 @@ _GetOrCreateShaderObject(
         return shaderObj;
     }
 
+    if (shadingNodeType == UsdMayaShadingNodeType::Shader) {
+        // We specifically reload only Pxr type shaders and let other importers
+        // work on unsupported nodes:
+        TfToken shaderId;
+        shaderSchema.GetIdAttr().Get(&shaderId);
+        if (_GetMayaTypeNameForShaderId(shaderId, false).IsEmpty()) {
+            return shaderObj;
+        }
+    }
```

Comment:
This still allows all utility nodes to be Maya nodes. Only surface shaders are checked here.